README.md

```diff
@@ -190,8 +212,34 @@ Note, some of the dependencies listed in the [initial setup](#initial-setup)
 may be necessary to install these rubies.  These dependencies will vary from
 platform to platform.
 
+### Running in development
+
+Although not necessary for deploy, you can also run zartan in development.
+If you do, there are a few additional steps.
+
 Create and modify the
-[config files](#config-files) for development.  Run `bundle exec rails s` to
+[config files](#config-files) for development.  Change directories to
+the root of zartan and run:
+```
+gem install bundler
+bundle install
+rake db:migrate
+```
+
+Unlike produciton, the values in
+[config/default_settings.yml](config/default_settings.yml) do not get
+automatically copied over.  You have to set them manually
```

Comment:
This process can be simplified. `rake config:seed` should work in dev, just like it does in production.